### PR TITLE
z/TPF Backtrace Translation and Processing Fixes

### PR DESCRIPTION
**Pull request by jjohnst-us**
z/TPF Symbol Names and Modules names from dladdr are
returned as EBCDIC characters sets.  These values need
to be translated before being stored in the backtrace
frame control structure otherwise they are printed out
as garbage characters.  Furthermore the top of stack for
z/TPF is a dummy invalid frame, and needs to be pruned
from the backtrace otherwise unexpected behaviour occurs.

Signed-off-by: James D Johnston <jjohnst@us.ibm.com>

### CODE REVIEW

File: port/ztpf/omrosbacktrace_impl.c [hunk 1]
```diff
@@ -246,6 +252,7 @@ omrintrospect_backtrace_symbols_raw(struct OMRPortLibrary *portLibrary,
 		uintptr_t symbol_offset = 0;
 		uintptr_t module_offset = 0;
 		const char *symbol_name = "";
+		char asciiSymbolBuffer[512], asciiModuleBuffer[512];
```
Comment (charliegracie) [inline port/ztpf/omrosbacktrace_impl.c]: Can you declare each variable on it's own line?

File: port/ztpf/omrosbacktrace_impl.c [hunk 2]
```diff
@@ -149,6 +151,10 @@ omrintrospect_backtrace_thread_raw(struct OMRPortLibrary *portLibrary,
 	ret = backtrace_sigprotect(portLibrary, threadInfo, addresses,
 		sizeof(addresses) / sizeof(void*));
 
+	//z/TPF Stack grows down.  The top-of-stack has a dummy frame which should be ignored.
+	//Therefore reducing stack frame count by one.
```
Comment (charliegracie) [inline port/ztpf/omrosbacktrace_impl.c]: `//` is not a proper C comment. Can you switch the comment to `/* */` style.

File: port/ztpf/omrosbacktrace_impl.c (hunk 1 above)
Comment (jjohnst-us) [reply port/ztpf/omrosbacktrace_impl.c]: Done.

File: port/ztpf/omrosbacktrace_impl.c (hunk 2 above)
Comment (jjohnst-us) [reply port/ztpf/omrosbacktrace_impl.c]: Done.

**Comment by charliegracie** [comment]
This is a z/tpf only file so no other CI builds are required.

**Pull request merged**